packages/node/src/models/state-channel.ts

```diff
@@ -540,7 +530,9 @@ export class StateChannel {
           return [appInstanceEntry[0], appInstanceEntry[1].toJson()];
         }
       ),
-      freeBalanceAppIndexes: Array.from(this.freeBalanceAppIndexes.entries()),
+      freeBalanceAppInstance: !!this.freeBalanceAppInstance
+        ? this.freeBalanceAppInstance.toJson()
+        : undefined,
```

Comment:
The free balance app instance should never be undefined.

Reply:
Before you run setup protocol though you technically do not have a free balance